release-notes/5.0/preview/api-diff/rc2/.Net/5.0-rc2.md

```diff
@@ -0,0 +1,11 @@
+# API Difference net5.0-rc1 vs net5.0-rc2
+
+API listing follows standard diff formatting. Lines preceded by a '+' are
+additions and a '-' indicates removal.
+
+* [System.Diagnostics](5.0-rc2_System.Diagnostics.md)
+* [System.Globalization](5.0-rc2_System.Globalization.md)
```

Comment:
Pipelines isn't mentioned in the summary.

Reply:
We should remove the pipelines changes between RC1 and RC2 because its basically a wash.

Reply:
It was removed from the shared framework, is that worth mentioning?

Reply:
should we remove the addition in rc1 as well ?